src/Sql/Sql/ChangeLog.md

```diff
@@ -18,41 +18,7 @@
         - Additional information about change #1
 -->
 ## Upcoming Release
-* Update Azure Paired Regions URL in Backup Storage Redundancy Warnings 
-
-## Version 2.11.0
-* Added BackupStorageRedundancy to the following: 
-    - `Restore-AzureRmSqlDatabase`
-    - `New-AzSqlDatabaseCopy`
-    - `New-AzSqlDatabaseSecondary`
-* Removed case sensitivity for BackupStorageRedundancy parameter for all SQL DB references 
-* Updated BackupStorageRedundancy warning message names
-* Added support for Managed HSM Uris for SQL DB and Managed Instance
-
-## Version 2.10.1
-* Added warning for BackupStorageRedundancy configuration in select regions in `New-AzSqlDatabase` (Ignore Case for BackupStorageRedundancy configuration input)
-* Fixed for bug in `New-AzSqlDatabaseExport`
-* Removed case sensitivity for BackupStorageRedundancy parameter for `New-AzSqlInstance` 
-
-## Version 2.10.0
-* Added BackupStorageRedundancy to `New-AzSqlInstance` and `Get-AzSqlInstance`
-* Added cmdlet `Get-AzSqlServerActiveDirectoryOnlyAuthentication`
-* Added cmdlet `Enable-AzSqlServerActiveDirectoryOnlyAuthentication`
-* Added Force parameter to `New-AzSqlInstance`
-* Added cmdlets for Managed Database Log Replay service
-	- `Start-AzSqlInstanceDatabaseLogReplay`
-	- `Get-AzSqlInstanceDatabaseLogReplay`
-	- `Complete-AzSqlInstanceDatabaseLogReplay`
-	- `Stop-AzSqlInstanceDatabaseLogReplay`
-* Added cmdlet `Get-AzSqlInstanceActiveDirectoryOnlyAuthentication`
-* Added cmdlet `Enable-AzSqlInstanceActiveDirectoryOnlyAuthentication`
-* Added cmdlet `Disable-AzSqlInstanceActiveDirectoryOnlyAuthentication`
-* Updated cmdlets `New-AzSqlDatabaseImport` and `New-AzSqlDatabaseExport` to support network isolation functionality
-* Added cmdlet `New-AzSqlDatabaseImportExisting`
-* Updated Databases cmdlets to support backup storage type specification
-* Added Force parameter to `New-AzSqlDatabase`
-* Updated ActiveDirectoryOnlyAuthentication cmdlets for server and instance to include ResourceId and InputObject
-* Added support for Managed HSM Uris for SQL DB and Managed Instance
+* Added DiffBackupIntervalInHours to `Set-AzSqlDatabaseBackupShortTermRetentionPolicy` 
```

Comment:
I just noticed that the change logs of past versions were deleted in this PR. Please be careful when you submit another PR. Thanks